bowling/inc/filesReader.hpp

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <experimental/filesystem>
+#include <memory>
+#include <string>
+
+#include "lane.hpp"
+
+namespace fs = std::experimental::filesystem;
+
+class FilesReader {
+public:
+    FilesReader(const std::string& directory);
+
+    size_t getLanesNum() const { return lanes_.size(); }
+    std::shared_ptr<Lane> getLane(size_t index) const;
+
+private:
+    fs::path directoryPath_;
+    std::vector<std::shared_ptr<Lane>> lanes_;
+
+    void checkDirectory() const;
+    std::vector<std::experimental::filesystem::path> makeFileList();
+    void readFiles(const std::vector<fs::path>& files);
+    void readPlayers(const std::experimental::filesystem::path& file, Lane& lane);
```

Comment:
As you defined `fs` namespace then you can use it here in lines 23 and 25, as you did in line 24.

But please take a look at my comment from here: https://github.com/kszytko/testing/pull/13#discussion_r481333055

> I'm not sure if defining namespace in .hpp files is a good practise? Wouldn't this namespace propagate to any other file where this .hpp is included? Do we want/need it?

Reply:
done